package.json

```diff
@@ -45,7 +45,7 @@
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
     "mocha": "^5.0.5",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.13.1",
```

Comment:
Could you help me understand the changes here, in relation to the commit message? I'm only seeing a change to the `package.json` file, but there's a mention of fixing vulnerabilities.

Reply:
The flagged vulnerabilities were fixed using `npm audit fix`, which essentially bumps the version for the affected dependencies.

Reply:
Ideally, we would have dependabot setup on these JavaScript repositories, but it seems to have been removed at some point in the past, so manual it is.